PCR - add buffering to remove gaps between census tracts ( fixes #814 )

nirajankeybridge opened the pull request:
Adds buffering to remove gaps between census tracts for PCR
Fixes https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/issues/814
Based on work by @chandanahs

kate-harrison commented:
Nirajan, thanks for this PR. So that we have the legal stuff squared away, can I ask if you have signed the contributor's agreement for the WinnForum repository? If not, please check with Lee Pucker and he can help you do that ASAP.

Dipti, may I ask which company you are with?

FYI test dev has a policy of requiring review from a SAS Admin in test dev *other than* the one which created the PR. I'll be happy to review this PR myself (and others are welcome to join if they wish) once the legal red tape is cleared.

nirajankeybridge commented:
Hi Kate, 
Thank you for checking and offering to review.
I have added the [Key Bridge Wireless CLA](https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/pull/852/commits/a3fc3dbe538d8655ead68f813365b5678472b103#diff-04667e7cacf31e35c6dc6f101e353c12) to this pull request as well. Lee reviewed the CLA and added me to the repository. Sorry for not including the CLA earlier.

nirajankeybridge commented:
Good catch, thanks Kate. I've added the [change to the pull request](https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/pull/852/commits/2e3bea255651bec476112d4222b554d28ed7185e)
